docs/user-docs/troubleshooting.md

```diff
@@ -77,7 +77,7 @@ can sometimes be necessary.
 
 This may be caused by the TPM safety timer.  If your machine was not safely shut
 down (e.g., due to a power loss) and it had been running for less than
-approximately 70 minutes (2<sup>22</sup> ms) since, you may experience such an
+approximately 70 minutes (2²² ms) since, you may experience such an
```

Comment:
Does CI complain about this? 2<sup>22</sup> isn't the same as 2²².

Reply:
force push with a mistake -> https://github.com/TrenchBoot/TrenchBoot.github.io/compare/b118d28d84f5942ed52a4177fd051642c23d367c..a692a4400a27df8fd548434840030742cacd8d46
 and immediate fix -> https://github.com/TrenchBoot/TrenchBoot.github.io/compare/a692a4400a27df8fd548434840030742cacd8d46..ca631218ba16e2b32aa7b9202bf6acb4608a21b7